tests/monitoring/monitoring.go

```diff
@@ -127,17 +128,28 @@ var _ = Describe("[Serial][sig-monitoring]Prometheus Alerts", func() {
 		BeforeEach(func() {
 			scales = make(map[string]*autoscalingv1.Scale, 1)
 			backupScale(virtOperatorDeploymentName)
+			backupScale(virtControllerDeploymentName)
 		})
 
 		AfterEach(func() {
+			revertScale(virtControllerDeploymentName)
 			revertScale(virtOperatorDeploymentName)
 			waitUntilThereIsNoAlert()
 		})
 
-		It("VirtOperatorDown should be triggered when virt-operator is down", func() {
+		It("VirtOperatorDown and NoReadyVirtOperator should be triggered when virt-operator is down", func() {
 			By("By scaling virt-operator to zero")
 			updateScale(virtOperatorDeploymentName, int32(0))
 			verifyAlertExist("VirtOperatorDown")
+			verifyAlertExist("NoReadyVirtOperator")
```

Comment:
Maybe we can introduce consts for alert names?

Reply:
done